### ISSUE TITLE
Images of different size and colour for selected and not selected items in the paging menu, animated.

### ISSUE THREAD

**Issue by IVANKRUPSKI** [kitasuke/PagingMenuController#310]
Hello,

Is it possible to do something similar to what is below in gif?

So, basically, to menu items add images of
1.  different colour for a selected and non-selected state, or alternatively one image but with ability to change it's tintColor depending on whether selected or not 

2. different size (or change size) for selected and not selected selected state.

And apply animation when moving form item to item.

Many thanks for the advice.
![example](https://cloud.githubusercontent.com/assets/16180201/21264664/a326cde2-c39d-11e6-83b1-58bfab55770b.gif)

**Comment by kitasuke** [comment]
Unfortunately it doesn't allow to apply the animation for now. What this library has is two images like below which are non selected one and selected one and It's simply switching depends on state.

```
public enum MenuItemDisplayMode {
    case text(title: MenuItemText)
    case multilineText(title: MenuItemText, description: MenuItemText)
    case image(image: UIImage, selectedImage: UIImage?)
    case custom(view: UIView)
}
```

**Comment by IVANKRUPSKI** [comment]
That's pity, would be really great to have animation.

Is there any chance you might make such an upgrade?
I can create separate request for that if necessary and test it later on.

Thank you very much for your help.

**Comment by AdAvAn** [comment]
I agree with @IVANKRUPSKI , this is an interesting functionality, it would be nice to implement it in PagingMenuController.

**Comment by kitasuke** [comment]
There used to be similar stuff, but I had some issues with AutoLayout and removed it... Maybe, I should try to implement this functionality again.

**Comment by IVANKRUPSKI** [comment]
That would be great!
